Depends-On: 0003_version_rewriter_test.patch
Touch-Files: src/unix/ibus/path_util.cc
Scope: path-isolation

diff --git a/src/unix/ibus/path_util.cc b/src/unix/ibus/path_util.cc
index 0000000..1111111 100644
--- a/src/unix/ibus/path_util.cc
+++ b/src/unix/ibus/path_util.cc
@@ -32,7 +32,7 @@
 #include <string>

 #ifndef MOZC_IBUS_INSTALL_DIR
-#define MOZC_IBUS_INSTALL_DIR "/usr/share/ibus-mozc"
+#define MOZC_IBUS_INSTALL_DIR "/usr/share/ibus-mozc4med"
 #endif  // MOZC_IBUS_INSTALL_DIR

 namespace mozc {
 namespace ibus {
